<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerMdUI_free_confirm</class>
 <widget class="QWidget" name="ManagerMdUI_free_confirm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>员工请假</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>请假理由</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_backtime">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>销假时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_reason">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Done">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
